### ISSUE TITLE
CST token listing request

### ISSUE THREAD

**Issue by KiddiK12** [forkdelta/tokenbase#583]
Contract address: 0x24ba8f48f225aed3abb6760fdce05c058e474c3e

Issuer's official website: https://cryptosocialnetwork.space/

Official contract address confirmation: https://cryptosocialnetwork.space/site/about

A link to a third-party review or discussion of the token: https://bitcointalk.org/index.php?topic=3195171

Description: CryptoSocialNetwork
CST launch was caused due to Google, Facebook and many other companies banned ICO and Crypto advertising in general.
CryptoSocialNetwork will use CST tokens instead of USD for bidding on advertising places and meanwhile you can enjoy chatting with people that have cryptorelated mindset.

**Comment by Kryptoniq** [comment]
Thanks for your request, we have verified that the information provided is accurate. We'll be listing the token in the next few days, you'll receive an update when we do.
0x24ba8f48f225aed3abb6760fdce05c058e474c3e, https://cryptosocialnetwork.space/

**Comment by freeatnet** [comment]
@KiddiK12 Can you post some more examples of project discussion? I can't seem to find anything but the already mentioned BitcoinTalk thread.

**Comment by KiddiK12** [comment]
Only BitcoinTalk thread at the moment. We dont want to create any social network channels as we literaly are social network.

In following weeks when we introduce Smart Bidding System we will offer bounty for articles on Medium and Steemit as SBS is the main reason network was created, due to facebook and google banned crypto related ads, SBS will allow users to use tokens and bid for advertising space, smart bot will allow them to set max. tokens used for advertisment so they wont even have to be arround as they will have personal bots bidding instead of them. According to programmers, SBS should be done within few weeks, till the end of the month max.

**Comment by KiddiK12** [comment]
Can we expect CST to be listed this week?

**Comment by freeatnet** [comment]
@KiddiK12 This isn't about social channels, more about lack of information or awareness of the project. Adding to the confusion, it appears a similarly named project launched publicly recently: "Minds: The Crypto Social Network" ([[1]], [[2]], [[3]])

I'd like to wait for some more publicity for your project, so that people purchasing your token can find out more about what they're buying into. @Kryptoniq, would you agree?

[1]: https://globenewswire.com/news-release/2018/03/28/1454618/0/en/Minds-Emerges-from-Beta-with-World-s-First-Crypto-Social-Network-for-Web-Android-and-iOS-on-Ethereum-Testnet-with-White-Paper.html
[2]: http://www.the-blockchain.com/2018/04/03/record-breaking-facebook-competitor-minds-launches-crypto-social-network/
[3]: https://github.com/Minds

**Comment by Kryptoniq** [comment]
@freeatnet Agreed. 
I also find it confusing to CryptoSocial.world (CSL). The only difference is the "network".

**Comment by KiddiK12** [comment]
@freeatnet @Kryptoniq  I must say i was not aware of "Minds: The Crypto Social Network" neither of CryptoSocial.world. We are not affiliated with them in any way, and we are not trying to copy other projects.

Another thing to mention, there is some wierd criteria to get listed arround here. WED got listed, just a quote from their website "If it's not Wednesday, don't count on it! We believe all business should be done on Wednesdays."

HELL coin quote: Satan was impressed with the blockchain technology and decided to make Hell Coin. This is the only currency that will be used in Hell – the place of selfish and awesome people. Hell Coin has undisputable advantages over the typical coins you are so used to.

    Makes you rich – Satanism will make you rich and powerful. It is a guarantee.
    Materialistic – Satan and his disciples are materialistic. They love embracing pleasures.
    Unmatchable speeds – Hell Coin gives you wings. No shit.

**Comment by KiddiK12** [comment]
To make things clear, im not trying to trash talk about other coins as im focused on our project, and CST is ready to comply with all ForkDelta rules.

That being sayd we just started bounty campaign where users got a task to spread the word via articles on various websites.

I just hope there will be no more delays in listing as we promised our users to get listed soon, since @Kryptoniq sayd we are good to go 18 days ago.

Kind regards
